src/Omega_h_mesh.hpp

```diff
@@ -125,26 +125,50 @@ class Mesh {
   Graph ask_star(Int dim);
   Graph ask_dual();
 
-  Adj ask_revClass (Int edim);
+/* ask_revClass (Int edim, LOs g_ids): takes input of entity dimension
+ * 'edim', and an 1d array of model entity IDs on local part to return
+ * a CSR structure containing IDs of mesh entities classified on the 
+ * requested input model entities
+ */
   Adj ask_revClass (Int edim, LOs g_ids);
+
+/* ask_revClass (Int edim): takes input of entity dimension 'edim' and
```

Comment:
same comment as `edim` for `ask_revClass` above

you can write 'see `ask_revClass (Int edim, LOs g_ids)`' then describe that all model entities are used instead of a subset/input list